src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
```

The SessionFactory is initialized eagerly at class-load time via the static initializer. That is fine, but it means class loading will fail early if hibernate.cfg.xml is missing or invalid. If you want to delay initialization until first use and avoid surprising classloader failures, consider a lazy-holder pattern or lazy initialization in getSessionFactory(). If you keep eager init, ensure hibernate.cfg.xml is present and correct (required by the task).